wasm_utils/src/api_serialization/get_entry.rs

```diff
@@ -44,31 +62,209 @@ pub struct GetEntryArgs {
     pub options: GetEntryOptions,
 }
 
-#[derive(Deserialize, Debug, Serialize, DefaultJson)]
+#[derive(Deserialize, Debug, Serialize, DefaultJson, Clone)]
+pub struct EntryResultMeta {
+    pub address: Address,
+    pub entry_type: EntryType,
```

Comment:
Why is the entry type part of meta? Entry type is also part of the entry now as per discussion with @artbrock and @thedavidmeister in Montreal and implemented by @thedavidmeister's "enum branch" recently. Do we need this here?

Reply:
As per verbal discussion: it is possible to set entry, header and sources to false in `GetEntryOptions`. That would return only the `EntryResultMeta` which then also carries the information of the entry type which otherwise can only be retrieved together with the entry. Getting the type of an potentially large entry wouldn't be possible if we hadn't this field here.